android/build.gradle.kts

```diff
@@ -23,6 +23,13 @@ android {
             proguardFiles(file("proguard-android-optimize.txt"), file("proguard-rules.pro"))
         }
     }
+
+    publishing {
+        singleVariant("release") {
+            withSourcesJar()
+            withJavadocJar()
```

Comment:
As mentioned in #32 this is going to be custom implemented to use dokka instead of actual javadocs.